assets/js/blocks/reviews-by-product/style.scss

```diff
@@ -34,121 +35,139 @@
 	}
 
 	.wc-block-reviews-by-product__orderby {
-		margin-bottom: $gap-larger;
+		margin-bottom: $gap-small;
 		text-align: right;
 	}
 
 	.wc-block-reviews-by-product__orderby__label {
 		margin-right: $gap-small;
+		display: inline-block;
+		font-weight: normal;
 	}
 
 	.wc-block-reviews-by-product__list {
-		list-style: none;
+		list-style: none outside;
```

Comment:
The initial value is `outside`, is there any need specifying it here?

Reply:
Again themes and to be safe it's set to this value. Some were showing bullets before.

Reply:
Still, `list-style` overwrites any previous declaration for the properties `list-style-type`, `list-style-image`, and `list-style-position`. So removing `outside` should be fine.